src/openrct2/world/Location.hpp

```diff
@@ -52,24 +52,30 @@ constexpr sint32 COORDS_NULL = -1;
  * Tile coordinates use 1 x/y increment per tile.
  * Regular ('big', 'sprite') coordinates use 32 x/y increments per tile.
  * */
-struct TileCoordsXY
+struct CoordsXY
```

Comment:
Why did you reverse the two? It only adds noise to the diff.

Reply:
Nick's new definitions in TileCoordsXY below require CoordsXY to be defined at an earlier point.

Reply:
The diff just looks confusing. `CoordsXY` has been moved to be above `TileCoordsXY`

Reply:
What @AaronVanGeffen said.